source/index.ts

```diff
@@ -109,49 +104,49 @@ function is(value: any): TypeName { // tslint:disable-line:only-arrow-functions
 }
 
 namespace is { // tslint:disable-line:no-namespace
-	const isObject = (value: any) => typeof value === 'object';
+	const isObject = (value: object) => typeof value === 'object'; // tslint:disable-line:strict-type-predicates
 
 	// tslint:disable:variable-name
 	export const undefined = isOfType<undefined>('undefined');
 	export const string = isOfType<string>('string');
 	export const number = isOfType<number>('number');
 	export const function_ = isOfType<Function>('function');
-	export const null_ = (value: any): value is null => value === null;
+	export const null_ = (value: null | any): value is null => value === null;
 	export const class_ = (value: any) => function_(value) && value.toString().startsWith('class ');
```

Comment:
Could this one have a `class` type?

See:
- https://stackoverflow.com/questions/39392853/is-there-a-type-for-class-in-typescript-and-does-any-include-it
- https://stackoverflow.com/questions/39976329/typescript-type-to-represent-any-class